src/spikeinterface/curation/auto_merge.py

```diff
@@ -61,9 +73,22 @@ def get_potential_auto_merge(
     ----------
     sorting_analyzer : SortingAnalyzer
         The SortingAnalyzer
-    minimum_spikes : int, default: 1000
+    preset : "default" | "lussac" | "temporal_splits" | "knn" | None, default: "default"
+        The preset to use for the auto-merge. Presets combine different steps into a recipe:
+        * "default" uses the following steps: "min_spikes", "remove_contaminated", "unit_positions",
+            "template_similarity", "correlogram", "check_increase_score"
+        * "lussac" uses the following steps: "min_spikes", "remove_contaminated", "unit_positions",
+            "template_similarity", "cross_contamination", "check_increase_score"
```

Comment:
what is the diff bewteen these two preset ?
not sure if lussac is the good naming.